include/veriblock/blockchain/pop/vbk_block_tree.hpp

```diff
@@ -19,9 +19,8 @@ struct VbkBlockTree : public BlockTree<VbkBlock, VbkChainParams> {
 
   VbkBlockTree(BtcTree& btc,
                std::shared_ptr<EndorsementsRepository> erepo,
```

Comment:
Shouldn't erepo be moved out as well?

Reply:
we use as read-only repo for fork resolution

Reply:
another possible way to solve it is to pass a function instead of repo, as we need only single function...